### ISSUE TITLE
lien externe en savoir plus ne fonctionne pas sur mobile

### ISSUE THREAD

**Issue by briceterdjman** [we-do-data/IFS_AFD-ACES-dev#45]
J'ai testé ce matin de rajouter http://wedodata.fr sur la card "Hello Oriane !". Le lien fonctionne sur desktop mais pas sur mobile.

**Comment by clementWedodata** [comment]
hello @JulienParis 

je me permet de te relancer sur ce ticket. Le projet va être montré en début de semaine prochaine à l'AFD, est-ce que tu as eu le temps de regarder ce problème ? 
Je te remercie.

**Comment by JulienParis** [comment]
j'avais mis en place un 'désactivateur' de ce bouton quand il n'y a pas de ressources (type url et textes)... sur mon mobile ce désactivateur marche normalement

**Comment by JulienParis** [comment]
une autre solution serait de faire 'disparaître' le bouton 'en savoir plus' ainsi que le texte s'il n'y a pas de ressource du tout

**Comment by JulienParis** [comment]
pardon j'ai un peu répondu à côté de la plaque... Je vais tester cette histoire de lien externe

**Comment by clementWedodata** [comment]
@JulienParis 

oui, d'un point de vue UX et accessibilité il vaut mieux masquer ce bouton si l'on a pas de ressources derrière. Je te laisse regarder cette histoire de lien qui ne fonctionne pas plus précisement ;)

**Comment by clementWedodata** [comment]
Merci

**Comment by JulienParis** [comment]
bon ... il y a un truc bizarre avec les boutons vuetify sur mobile : sur mobile un bouton-icône va réagir à peu près normalement (genre le bouton 'favoris'), mais un lien (balise <a>) ou un bouton contenant du texte va se geler en mode 'hover' et ne pige pas qu'il a été cliqué...

Le petit hack en attente de trouver mieux : sur mobile uniquement, j'ai enlevé le surlignage du lien et ajouté un petit icône d'hyperlien à côté du texte - qui lui fonctionne et renvoit vers le lien... Pas top mais c'est en attendant de trouver quelle partie du CSS des boutons ignore l'évènement 'click' sur mobile... :/
